File: 프로그래머스/unrated/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    col = []
    row = []
    
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[0])):
            if wallpaper[i][j] == "#":
                col.append(i)
                row.append(j)
    
    return [min(col), min(row), max(col) + 1, max(row) + 1]

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    const xPositions = [];
    const yPositions = [];
    for (let i = 0; i < wallpaper.length; i++) {
        for (let j = 0; j < wallpaper[0].length; j++) {
            if (wallpaper[i][j] === "#") {
                xPositions.push(i);
                yPositions.push(j);
            }
        }
    }
    return [Math.min(...xPositions), Math.min(...yPositions), Math.max(...xPositions) + 1, Math.max(...yPositions) + 1];
}